app/src/main/res/layout/drag_drop_interaction_items.xml

```diff
@@ -20,27 +20,67 @@
     android:layout_margin="4dp"
     android:orientation="horizontal">
 
-
-    <androidx.recyclerview.widget.RecyclerView
-      android:id="@+id/drag_drop_item_recyclerview"
+    <LinearLayout
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="8dp"
       android:layout_marginTop="@dimen/divider_margin_top"
       android:layout_marginEnd="8dp"
-      android:layout_weight="1"
       android:background="@drawable/rounded_rect_grey_border_white_background"
       android:paddingStart="16dp"
       android:paddingEnd="16dp"
-      app:itemDecorator="@{@drawable/divider}"
-      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toStartOf="@+id/drag_drop_content_group_item"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"
-      app:list="@{viewModel.htmlContent.htmlList}"
-      tools:itemCount="3"
-      tools:listitem="@layout/drag_drop_single_item" />
+      app:layout_constraintTop_toTopOf="parent">
+
+      <LinearLayout
+        android:id="@+id/drag_drop_accessible_container"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="8dp">
+
+        <ImageButton
+          android:id="@+id/drag_drop_move_up_item"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:background="?attr/selectableItemBackground"
+          android:clickable="true"
+          android:contentDescription="@string/move_item_up"
+          android:enabled="@{viewModel.itemIndex != 0}"
+          android:focusable="true"
+          android:onClick="@{(v) -> viewModel.handleUpMovement(adapter)}"
+          android:src="@drawable/ic_keyboard_arrow_down_black_24dp"
```

Comment:
This icon would be replaced with up arrow once I get it from Chantel @rt4914 @BenHenning